observing_examples.ipynb

```diff
@@ -0,0 +1,971 @@
G1PDQAdSzQpqOyV3yqn\nvtbKYTn16dm2z0oKnTQpqCxWWV7idQjKYZnsAp5bSWGvlRTKNCkopQLk5KFWleDYvu5XJ+VcUijM\nb6UjmpXjTh7axesQVBYLDafIxJT/OZUUtu6poVNJgU5+pRz3wKVVXoegUhC6Ehw/yH8TLYZfpyrK\nioFDJQY35VSH/e37arSRWbkiF280RvRq73UIaWvVSnjjpol0a1/UZHvPDm1Yt2O/R1E117usmE9u\nP4V2GVhGOLdKCntrtD1Bpe3uc4d7HYLnOpUUcMvpQ70OwxGV5SUUBWChoPZtWmfk5iOnksK2vQe1\n55FKW2gcQC6v7TD3J5MY109HlWejnKo+2ranhrISnd1TpadT20JWTz+DnftrGXHnK43bH796HG20\nE4NySIlHszm7WlIQkckislRElovItCjPF4rIP+3nPxCRSrdi2XWglr019Trls3LN+P6dOLJ3B6/D\nUA644aQBXofAtNO8qZ5zLSmISB5wH3AaMAy4WESGRex2JbDdGDMA+C1wr1vxLN2wG7CGjivliOBP\niKliuGh0hdcheLb2tZslhTHAcmPMSmNMDfAEcHbEPmcDj9iPnwZOEpdaUt5YugmAod3buXF4lYMk\nh1rkXvn+cV6HoDLEzVTUE1gT9vtaYGysfYwxdSKyE+gEbHE6mMvGV1LetrBZ1zOlUtWuKHdmau2S\nI9Wu150wgHdXOH75SdhPzzmcjTsPcPWEfp7FIMallYJE5AJgsjHmKvv3bwBjjTHXhe3zqb3PWvv3\nFfY+WyKOdQ1wDUBFRcWozz//3JWYlUpW6Pvjx3EKldNeSPo15x3Vk+4divjfN1YQujTce/5wLhpd\nwfa9NSxev4ueHdvQp1P2z6/09rItTH9pMZ+u2+X6uaafN5zdB+q4+jj3koGIzDXGxB1l6WZSGA/c\nYYw51f79FgBjzD1h+7xs7/OeiOQDG4DOpoWgqqqqTHV1tSsxK6VUtko0KbhZKzoHGCgifUWkAJgC\nzIjYZwZwmf34AuD1lhKCUkopd7nWpmC3EVwHvAzkAQ8ZYxaKyF1AtTFmBvAg8HcRWQ5sw0ocSiml\nPOJqnydjzExgZsS228IeHwC+6mYMSimlEpdDneqUUkrFo0lBKaVUI00KSimlGmlSUEop1UiTglJK\nqUauDV5zi4hsBqINaS7HhekxXKYxZ0bQYg5avKAxZ0o6MfcxxsRdBCRwSSEWEalOZLSen2jMmRG0\nmIMWL2jMmZKJmLX6SCmlVCNNCkoppRplU1K43+sAUqAxZ0bQYg5avKAxZ4rrMWdNm4JSSqn0ZVNJ\nQSmlVJp8mxREpLeIzBaRRSKyUES+Z2+/Q0TWicg8++f0sNfcIiLLRWSpiJwatn2yvW25iExzOe7V\nIrLAjq3a3lYmIrNEZJn9b0d7u4jIH+y45ovIyLDjXGbvv0xELot1PgfiHRz2Xs4TkV0icqPf3mcR\neUhENtkLM4W2Ofa+isgo+/9tuf3atFfNiRHzL0VkiR3XsyLSwd5eKSL7w97vP8eLLdbf70LMjn0W\nxJpK/wN7+z/Fmlbf6Xj/GRbrahGZZ2/3y3sc69rmj8+zMcaXP0B3YKT9uBT4DBgG3AHcFGX/YcAn\nQCHQF1iBNWV3nv24H1Bg7zPMxbhXA+UR234BTLMfTwPutR+fDrwICDAO+MDeXgastP/taD/umIH3\nPA9roaM+fnufgeOAkcCnbryvwIf2vmK/9jSXYj4FyLcf3xsWc2X4fhHHiRpbrL/fhZgd+ywATwJT\n7Md/Bq51Ot6I538N3Oaz9zjWtc0Xn2fflhSMMeuNMR/Zj3cDi7HWdI7lbOAJY8xBY8wqYDkwxv5Z\nboxZaYypAZ6w982ks4FH7MePAOeEbX/UWN4HOohId+BUYJYxZpsxZjswC5icgThPAlYYY1pa79ST\n99kY8xbWmhuRsaT9vtrPtTPGvG+sb9SjYcdyNGZjzCvGmDr71/eBXi0dI05ssf5+R2NuQVKfBftu\n9UTgaadibile+3wXAo+3dAwP3uNY1zZffJ59mxTCiUglcBTwgb3pOrsY9VBYca4nsCbsZWvtbbG2\nu8UAr4jIXLHWlgboaoxZbz/eAHS1H/sl5pApNP0C+fl9Bufe157248jtbvsm1l1cSF8R+VhE3hSR\nCfa2lmKL9fe7wYnPQidgR1hSdPt9ngBsNMYsC9vmq/c44trmi8+z75OCiLQFngFuNMbsAv4E9AeO\nBNZjFQ/95FhjzEjgNOC7InJc+JN25vZdly+7bvcs4Cl7k9/f5yb8+r7GIiI/BuqAf9ib1gMVxpij\ngKnAYyLSLtHjufz3B+qzEOZimt7k+Oo9jnJtc+1cyfB1UhCR1lhv2j+MMf8CMMZsNMbUG2MagAew\niqoA64DeYS/vZW+Ltd0Vxph19r+bgGft+DbaRbpQUXWTn2K2nQZ8ZIzZCP5/n21Ova/raFqN42rs\nInI58BXg6/aXH7sKZqv9eC5WnfygOLHF+vsd5eBnYStW1Ud+xHbH2ec4D/hn2N/hm/c42rWthXNl\n9PPs26Rg1wc+CCw2xvwmbHv3sN3OBUK9DmYAU0SkUET6AgOxGlvmAAPtXg8FWFUkM1yKuURESkOP\nsRoVP7XPF+oZcBnwXFjMl9q9C8YBO+3i48vAKSLS0S6qn2Jvc1OTuyo/v89hHHlf7ed2icg4+3N3\nadixHCUik4EfAmcZY/aFbe8sInn2435Y7+vKOLHF+vudjtmRz4KdAGcDF7gdM3AysMQY01iN4pf3\nONa1rYVzZfbznGiLdKZ/gGOxik/zgXn2z+nA34EF9vYZQPew1/wYK/svJay13X7dZ/ZzP3Yx5n5Y\nPS0+ARaGzoVVl/oasAx4FSiztwtwnx3XAqAq7FjfxGq4Ww5c4fJ7XYJ1F9c+bJuv3meshLUeqMWq\nI73SyfcVqMK62K0
```

Comment:
Maybe instead of `synphot` you can write "predicted" counts here? Your choice.

Reply:
Or best of both worlds: "counts predicted by synphot"?